client/src/main/java/io/pravega/client/connection/impl/RawClient.java

```diff
@@ -59,6 +60,8 @@ public void process(Reply reply) {
                 }
             } else if (reply instanceof WireCommands.WrongHost) {
                 closeConnection(new ConnectionFailedException(reply.toString()));
+            } else if (reply instanceof WireCommands.ErrorMessage) {
+                errorMessage((ErrorMessage) reply);
```

Comment:
This is not going to propagate up to the caller.

Reply:
I might be misunderstanding, but it seems to propogate properly on like 97 in `RawClientTests`.